app/controllers/api/v1/countries_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::CountriesController < ApplicationController
+  def show
+    render json: Country.find(country_id)
+  end
+
+  private
+
+    def strong_params
+      params.permit(:id)
+    end
+
+    def country_id
+      strong_params["id"]
```

Comment:
I like the attempt to refactor here. Can we go with `country_params` instead of `strong_params`? Then your find could be `Country.find(country_params[:id])` which would be just as clear with a few less lines to maintain.